packages/components/src/Components/Truncate/Truncate.js

```diff
@@ -1,13 +1,12 @@
-import React from 'react';
-import propTypes from 'prop-types';
-
-import classNames from 'classnames';
-
-import { Button, Stack, StackItem } from '@patternfly/react-core';
-
 import './truncate.scss';
 
-import sanitizeHtml from 'sanitize-html';
+import { Stack, StackItem } from '@patternfly/react-core/dist/js/layouts/Stack/index';
+
+import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
+import React from 'react';
+import classNames from 'classnames';
+import propTypes from 'prop-types';
+import sanitizeHtml from 'sanitize-html/dist/sanitize-html';
```

Comment:
I am not sure if this is going to help. The module main attribute (in package.json) points to `sanitize-html/dist/sanitize-html` anyway.

This is thing is simply HUGE.

Reply:
maybe we should explicitly point to the minned one 🤔 half the size
<img width="635" alt="Screen Shot 2020-02-13 at 8 52 05 AM" src="https://user-images.githubusercontent.com/6640236/74441664-3dcceb00-4e3e-11ea-9cae-73f030698775.png">

Reply:
It should get minimized by webpack while building the bundles